File: app/src/main/java/me/toptas/rssconvertersample/MainActivity.java
package me.toptas.rssconvertersample;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        getSupportFragmentManager()
                .beginTransaction()
                .add(R.id.rlContainer, RssFragment.newInstance("https://www.wired.com/feed/"))
                .commit();


    }
}

File: app/src/main/java/me/toptas/rssconvertersample/MainActivity.kt
package me.toptas.rssconvertersample

import android.os.Bundle
import android.support.v7.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        supportFragmentManager
                .beginTransaction()
                .add(R.id.rlContainer, RssFragment.newInstance("https://www.wired.com/feed/"))
                .commit()


    }
}
